### DISCUSSION TITLE
Why can't we Open link in new tab in Github?

### DISCUSSION THREAD

**Discussion by saikumarb12345** [EddieHubCommunity/support#7320] [Q&A] [answered]
Why can't we Open link in new tab in Github?

**Comment by Panquesito7** [comment]
What do you mean by this exactly? 🤔

**Reply by saikumarb12345** [reply]
First of all thanks for your reply,
Yes,
When we click any links in github, it's opening in the same tab rather thatnopening in new tab....

**Reply by Panquesito7** [reply]
I think that's how GitHub works. In case you want it to be changed, you should [contact GitHub](https://support.github.com/).

**Comment by gekkowrld** [comment]
**This is my opinion but may be the case:**
Opening links in new tab can clutter the browser window. Many people open links and never bother to close the tabs after finishing what they wanted to do. This puts pressure on the CPU and may cause the device to slow down. So to avoid this problem they decided that most links will be opened in the same tab.

**Reply by EmmaDawsonDev** [reply, marked as answer]
This is how links are supposed to work. They're meant to open in the same tab and from an accessibility perspective it's good practice because it's less disorienting for people who have low vision/can't see where the new tab opens

**Comment by MaorBemdoo** [comment]
Good day, I'm Bemdoo:
My opinion from my knowledge of HTML is probably that all links do not contain the `<a href="" target="_blank"></a>`
The target attribute opens a new tab when a link is clicked

**Comment by sachasa** [comment]
GitHub's default behavior for links is to open them in the same tab or window. This is a deliberate design choice made by GitHub to maintain consistency in the user experience and prevent cluttering users' browsers with numerous open tabs.

GitHub's decision to keep links opening in the same tab is based on several factors:

User Experience Consistency: By default, most web applications and websites follow the convention of opening links in the same tab to avoid overwhelming users with multiple tabs.

Controlled Navigation: Opening links in the same tab ensures that users stay within the GitHub interface, making it easier for them to navigate back to the previous page.

Predictability: Users generally expect links within an application or website to open in the same tab unless they choose to open them in a new tab manually (by right-clicking or using keyboard shortcuts).

While GitHub doesn't provide a built-in option to open links in new tabs within its interface, users can still choose to open links in new tabs manually by right-clicking on the link and selecting "Open link in new tab" from the context menu or by using keyboard shortcuts (e.g., Ctrl+click or Cmd+click).

However, it's worth noting that GitHub may introduce changes to its user interface and functionality over time, so it's a good idea to check their documentation or announcements for any updates regarding link behavior or other features.

**Comment by MehmetBozkir** [comment]
This is a rule set by github for ease of use. 

If you are very uncomfortable about this issue, you can solve your problem by assigning a macro to one of the mouse buttons.
